app/src/main/java/com/example/newsapp/ui/home/viewmodels/EditProfileViewModel.kt

```diff
@@ -64,7 +64,7 @@ class EditProfileViewModel @Inject constructor(
     fun updateUser(fullName: String) {
         uiScope.launch {
             val user = repository.getUser(userEmail)
-            user?.let{
+            user?.let {
                 if (!fullName.isNullOrEmpty()) {
```

Comment:
@jnandrawos inverted if condition, try using the true value without negating it first and no need for null here since fullName is non-nullable you can use isEmpty or isBlank instead